api/python/quilt3/packages.py

```diff
@@ -12,6 +13,12 @@
 import warnings
 
 import jsonlines
+from tqdm import tqdm
+
+try:
+    import ujson as json  # If you have ujson, JSONL parsing is slightly faster, but is not a required dependency
+except ImportError:
+    pass
```

Comment:
You should move `import json` here, instead of importing it early and overwriting it